moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -95,6 +117,15 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
     }
   }
 
+  const Eigen::MatrixXd goal_state = trajectory.getTrajectoryPoint(goal_index);
```

Comment:
why ``Xd`` instead of Affine3d like we normally use in moveit?

Reply:
That's what the `ChompTrajectory` API uses.